.github/workflows/build.yml

```
@@ -41,48 +41,88 @@
-        run: dotnet-coverage merge "coverage.xml" "dotnet-coverage collect \"dotnet test VictoryCenter/VictoryCenter.IntegrationTests/VictoryCenter.IntegrationTests.csproj --configuration Release\" -f xml -o \"coverage2.xml\"" -o "coverage.xml"
+      - name: Show first 10 lines of coverage.xml
+        run: head -n 10 CoverageReport/Cobertura.xml
 
       - name: End SonarCloud analysis
         env:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         run: |
-          .\.sonar\scanner\dotnet-sonarscanner end /d:sonar.token="${{ secrets.SONAR_TOKEN }}"
+          ./.sonar/scanner/dotnet-sonarscanner end \
+          /d:sonar.token="${{ secrets.SONAR_TOKEN }}" \
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)